packages/instantsearch-react/src/translatable.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export default function translatable(defaultTranslations) {
+  return Composed => {
+    function Translatable(props) {
+      const {translations, ...otherProps} = props;
+      const translate = (key, ...params) => {
+        const translation =
+          translations && {}.hasOwnProperty.call(translations, key) ?
```

Comment:
Well for all the stupid stuff anyone can do with javascript, I wouldn't write that this way. What is the use case of `Object.create(null)`? Will it apply in our case? If it's not likely, I wouldn't use that... Sanity of the code first because we'll have to maintain it.